cdap-relational-spi/src/main/java/io/cdap/cdap/etl/spi/relational/ExpressionConversionContext.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.etl.spi.relational;
+
+import com.google.inject.Inject;
+import io.cdap.cdap.api.data.schema.Schema;
+
+import javax.annotation.Nullable;
+
+/**
+ * A class that provides a Context object to facilitate working with {@link SqlDialectConversionExpressionFactory}
+ * objects with specific SQL dialects
+ */
+public class ExpressionConversionContext {
```

Comment:
This class should be moved out of the SPI module . Only keep the API interface and proto style objects/exception included in the interfaces.

Reply:
+1